storagetransfer/check_latest_transfer_operation.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Command-line sample that checks the latest transfer operation for a given
+transfer job.
+"""
+
+
+import argparse
+
+# [START storagetransfer_get_latest_transfer_operation]
+from google.cloud import storage_transfer
+
+
+def check_latest_transfer_operation(project_id: str, job_name: str):
+    """Checks the latest transfer operation for a given transfer job."""
+
+    client = storage_transfer.StorageTransferServiceClient()
+
+    # The ID of the Google Cloud Platform Project that owns the job
+    # project_id = 'my-project-id'
+
+    # Storage Transfer Service job name
+    # job_name = 'transferJobs/1234567890'
+
+    transfer_job = client.get_transfer_job({
+        'project_id': project_id,
+        'job_name': job_name,
+    })
+
+    if transfer_job.latest_operation_name:
+        transfer_operation = storage_transfer.TransferOperation(
+            name=transfer_job.latest_operation_name, project_id=project_id)
```

Comment:
It looks like this just creates a local TransferOperation object with the name we got from the transfer job. We actually want to be able to see info like the job's status, so it should be a service call that gets the latest transfer operation based on the name